functions/Get-DbaAgListener.ps1

```diff
@@ -0,0 +1,87 @@
+function Get-DbaAgListener {
+    <#
+        .SYNOPSIS
+            Outputs the name of the Listener for the Availability Group(s) found on the server.
+
+        .DESCRIPTION
+            Default view provides most common set of properties for information on the database in an Availability Group(s).
+
+            Information returned on the database will be specific to that replica, whether it is primary or a secondary.
+
+            This command will return an SMO object, but it is the AvailabilityDatabases object  and not the Server.Databases object.
+
+        .PARAMETER SqlInstance
+            The SQL Server instance. Server version must be SQL Server version 2012 or higher.
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted).
+
+        .PARAMETER AvailabilityGroup
+            Specify the Availability Group name that you want to get information on.
+
+        .PARAMETER Silent
```

Comment:
Just so we keep with our standard can you go ahead and add a Listener parameter.

I doubt it will be used but we offer a filter for the object the command interacts with so might as well add that to this one.